src/libs/actions/Report.js

```diff
@@ -219,7 +209,8 @@ function getSimplifiedIOUReport(reportData, chatReportID) {
         stateNum: reportData.stateNum,
         submitterPayPalMeAddress: reportData.submitterPayPalMeAddress,
         submitterPhoneNumbers: reportData.submitterPhoneNumbers,
-        hasOutstandingIOU: reportData.stateNum === 1 && reportData.total !== 0,
+        hasOutstandingIOU: reportData.stateNum === CONST.REPORT.STATE_NUM.PROCESSING && reportData.total !== 0,
```

Comment:
NAB I still can't help but feel that these two properties should be combined into one. I think a report can be in any of these states:
1. There is no IOU
2. There is an outstanding IOU
3. There is an IOU that has been paid

I'm not sure why anything cares about `3`. If there is indeed a valid case, then I think using something more like the `stateNum` (disclaimer: I have no idea what `stateNum` is for) would be better.

Reply:
Yep, I think for now `hasOutstandingIOU` is sufficient. Will revert this change.